<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeaderPage</class>
 <widget class="QWidget" name="HeaderPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
 border:2px solid white;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px &quot;微软雅黑&quot;;
background-color: rgb(115, 115, 115);
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>网压</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string>网流</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string>当前站</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Voltage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Current">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_CurStation">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Date">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Time">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_TrainNum">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>909</x>
     <y>2</y>
     <width>121</width>
     <height>96</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/images/fault.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_PagePosition">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string>运行页面</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="headerlabel_Runmode">
   <property name="geometry">
    <rect>
     <x>749</x>
     <y>1</y>
     <width>160</width>
     <height>98</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(115, 115, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
